src/NHPersistenceCompatibilityTests/Common/NugetHelper.cs

```diff
@@ -27,10 +30,25 @@ public IEnumerable<string> GetPossibleVersionsFor(string packageName, string min
 
         internal void DownloadPackageTo(string packageName, string version, string location)
         {
-            var repo = PackageRepositoryFactory.Default.CreateRepository(packageSource);
+            try
+            {
+                InstallPackageFromSource(packageSource, packageName, version, location);
+            }
+            catch (Exception e)
```

Comment:
@HEskandari this is caused by the fact that for beta-0003 `NServiceBus.NHiberante` is on nuget but `NServiceBus` dependency is only on myget. Unfortunately the exceptions throw here are very generic :/